crates/cairo-addons/src/vm/hint_definitions/hashdict.rs

```diff
@@ -232,6 +233,44 @@ pub fn get_keys_for_address_prefix() -> Hint {
     )
 }
 
+pub fn hashdict_read_from_key() -> Hint {
```

Comment:
maybe not worth it but should we diff test rust hints vs. python hints in some 'devx' continuous improvement?